python-sdk/pachyderm_sdk/api/pps/_additions.py

```diff
@@ -0,0 +1,10 @@
+from . import CreatePipelineRequest, Service, Spout
```

Comment:
Could you copy the structure of file docstring from the `pfs/_additions.py` file?